docs/OV_Runtime_UG/supported_plugins/GPU.md

```diff
@@ -297,12 +295,12 @@ The behavior depends on specific parameters of the operations and hardware confi
 
 
 ## GPU Performance Checklist: Summary <a name="gpu-checklist"></a>
-Since the OpenVINO relies on the OpenCL&trade; kernels for the GPU implementation. Thus, many general OpenCL tips apply:
--	Prefer `FP16` inference precision over `FP32`, as the Model Optimizer can generate both variants and the `FP32` is default. Also, consider [int8 inference](@ref openvino_docs_model_optimization_guide).
-- 	Try to group individual infer jobs by using [automatic batching](../automatic_batching.md)
--	Consider [caching](../Model_caching_overview.md) to minimize model load time
--	If your application is simultaneously using the inference on the CPU or otherwise loads the host heavily, make sure that the OpenCL driver threads do not starve. You can use [CPU configuration options](./CPU.md) to limit number of inference threads for the CPU plugin.
--	Even in the GPU-only scenario, a GPU driver might occupy a CPU core with spin-looped polling for completion. If the _CPU_ utilization is a concern, consider the dedicated referenced in this document. Notice that this option might increase the inference latency, so consider combining with multiple GPU streams or [throughput performance hints](../performance_hints.md).
+Since OpenVINO relies on the OpenCL&trade; kernels for the GPU implementation, many general OpenCL tips apply:
+-	Prefer `FP16` inference precision over `FP32`, as Model Optimizer can generate both variants and the `FP32` is the default. Also, consider [int8 inference](../Int8Inference.md).
+- Try to group individual infer jobs by using [automatic batching](../automatic_batching.md).
+-	Consider [caching](../Model_caching_overview.md) to minimize model load time.
+-	If your application performs inference on the CPU alongside the GPU, or otherwise loads the host heavily, make sure that the OpenCL driver threads do not starve. You can use [CPU configuration options](./CPU.md) to limit the number of inference threads for the CPU plugin.
+-	Even in the GPU-only scenario, a GPU driver might occupy a CPU core with spin-looped polling for completion. If CPU load is a concern, consider the dedicated `queue_throttle` property mentioned previously. Notice that this option may increase inference latency, so consider combining with multiple GPU streams or [throughput performance hints](../performance_hints.md).
```

Comment:
```suggestion
Since OpenVINO relies on the OpenCL&trade; kernels for the GPU implementation, many general OpenCL tips apply:
- Prefer `FP16` inference precision over `FP32`, as Model Optimizer can generate both variants and the `FP32` is the default. Also, consider [int8 inference](../Int8Inference.md).
- Try to group individual infer jobs by using [automatic batching](../automatic_batching.md).
- Consider [caching](../Model_caching_overview.md) to minimize model load time.
- If your application performs inference on the CPU alongside the GPU, or otherwise loads the host heavily, make sure that the OpenCL driver threads do not starve. You can use [CPU configuration options](./CPU.md) to limit the number of inference threads for the CPU plugin.
- Even in the GPU-only scenario, a GPU driver might occupy a CPU core with spin-looped polling for completion. If CPU load is a concern, consider the dedicated `queue_throttle` property mentioned previously. Notice that this option may increase inference latency, so consider combining with multiple GPU streams or [throughput performance hints](../performance_hints.md).
```